### PR TITLE
Fix #115: added test cases for sentiment analysis

### LINKED ISSUE #115: Create Test for sentiment analysis of a blog.
Need to write test for code in https://github.com/Seneca-CDOT/telescope/blob/master/src/sentiment-analysis.js file to check its functionality.

I will take up this issue but will need help as I have not written tests from scratch. Also, other pull request are welcomed for fixing this issue.

### PR DESCRIPTION
**Pull request by dbeigi**
This pull request is to address #115 . I wrote two test cases that will test if sentiment analysis functionality will a) return the unexpected object b) return the expected object.

@jatinAroraGit would love to see your feedback on the following tests.

### CODE REVIEW

File: test/sentiment-analysis.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    'love',
+  ],
+  negative: [
+    'allergic',
+  ],
+};
+
+test('test if sample text does not return the correct object', () => sentimentAnalysis.run('I like dogs, but my girlfriend is afraid of them').then((data) => {
+  expect(data).toEqual(expect.not.objectContaining(expected));
+}));
```
Comment (manekenpix) [inline test/sentiment-analysis.test.js]: Just a suggestion:
```js
test('test if sample text does not return the correct object', async () => {
  const data = await sentimentAnalysis.run('I like dogs, but my girlfriend is afraid of them');
  expect(data).toEqual(expect.not.objectContaining(expected));
});
```

File: test/sentiment-analysis.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+test('test if sample text returns the correct object', () => sentimentAnalysis.run('I love cats, but I am allergic to them.').then((data) => {
+  expect(data).toEqual(expect.objectContaining(expected));
+}));
```
Comment (manekenpix) [inline test/sentiment-analysis.test.js]: Same here
```js
test('test if sample text returns the correct object', async () => {
  const data = await sentimentAnalysis.run('I love cats, but I am allergic to them.');
  expect(data).toEqual(expect.objectContaining(expected));
});
```

Comment (dbeigi) [reply test/sentiment-analysis.test.js]: applied the requested change.

File: test/sentiment-analysis.test.js (hunk 1 above)
Comment (dbeigi) [reply test/sentiment-analysis.test.js]: applied the requested change.

File: test/sentiment-analysis.test.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    'love',
+  ],
+  negative: [
+    'allergic',
+  ],
+};
+
+test('test if sample text does not return the correct object', async () => {
+  const data = await sentimentAnalysis.run('I like dogs, but my girlfriend is afraid of them');
+  expect(data).toEqual(expect.not.objectContaining(expected));
```
Comment (humphd) [inline test/sentiment-analysis.test.js]: Why do you not just compare `data` to `expected` with a deep equals here?  What does `.toEqual(expect.not.objectContaining(expected))` mean?  Maybe you're right, but this seems odd.

Comment (dbeigi) [reply test/sentiment-analysis.test.js]: @humphd From what I have read, `.toEqual` should be used for objects. Regards to `expect.not.objectContaining(expected)` , I am using this to check if the contents(properties) of the object do not match what we are expecting. This was suggested by @jatinAroraGit and after reading the docs I agreed to his suggestion.

Comment (humphd) [reply test/sentiment-analysis.test.js]: Interesting, OK, sounds fine.  Thanks for educating me.

**Pull request merged**